File: src/main/java/hu/kirdev/schpincer/web/comonent/CustomComponentAnswer.java
package hu.kirdev.schpincer.web.comonent;

import java.util.List;

import lombok.Data;

@Data
public class CustomComponentAnswer {

    private String type;
    private String name;
    private List<Integer> selected;
    
    @Data
    public static class CustomComponentAnswerList {
        List<CustomComponentAnswer> answers;
    }
    
}

File: src/main/kotlin/hu/kirdev/schpincer/web/component/CustomComponentAnswer.kt
package hu.kirdev.schpincer.web.component

data class CustomComponentAnswer(
        val type: String = "",
        val name: String = "",
        val selected: List<Int> = listOf()
)

data class CustomComponentAnswerList(
        val answers: List<CustomComponentAnswer> = listOf()
)
